lib/model/model.go

```diff
@@ -657,12 +672,13 @@ nextFolder:
 			l.Infof("Unexpected folder ID %q sent from device %q; ensure that the folder exists and that this device is selected under \"Share With\" in the folder configuration.", folder.ID, deviceID)
 			continue
 		}
+		sharedFolders = append(sharedFolders, folder.ID)
 	}
 	m.fmut.Unlock()
 
-	events.Default.Log(events.DeviceConnected, event)
-
-	l.Infof(`Device %s client is "%s %s" named "%s"`, deviceID, cm.ClientName, cm.ClientVersion, cm.DeviceName)
+	if cm.Flags&protocol.FlagClusterConfigTemporaryIndexesDisabled == 0 {
+		m.progressEmitter.temporaryIndexSubscribe(conn, sharedFolders)
```

Comment:
Comment that this breaks if we get more than one cluster config per connection, as we talk about that above